policyengine_us/tests/policy/baseline/household/income/spm_unit/spm_unit_broadband_subsidy.yaml

```diff
@@ -13,4 +13,22 @@
     acp: 2
     ebb: 3
   output:
-    spm_unit_broadband_subsidy: 6
+    spm_unit_broadband_subsidy: 5
+
+- name: Otherwise sum ACP 1, EBB 1
+  period: 2023
+  input:
+    gov.simulation.reported_broadband_subsidy: false
+    acp: 1
+    ebb: 2
+  output:
+    spm_unit_broadband_subsidy: 3
+
+- name: Otherwise sum ACP 0, EBB 3
+  period: 2023
+  input:
+    gov.simulation.reported_broadband_subsidy: false
+    acp: 1
+    ebb: 1
+  output:
+    spm_unit_broadband_subsidy: 2
```

Comment:
```suggestion
```

Dont need to add another test - one baseline unit test is fine - there is no significant other edge case to test